lib/src/base-styles/_nx-variables.scss

```
@@ -62,9 +62,9 @@
   --nx-border-grid: var(--nx-border-2x);
 
   // for dropdown
-  --nx-box-shadow-dropdown: 0 2px 12px 0 rgba(74, 77, 113, 0.2),
-                            0 1px 2px 0 rgba(91, 95, 141, 0.2),
-                            0 1px 1px 0 rgba(124, 128, 179, 0.2);
+  --nx-box-shadow-dropdown: 0 2px 12px 0 Hsla(231, 43%, 80%, 0.2),
```

```suggestion
  --nx-box-shadow-dropdown: 0 2px 12px 0 Hsla(var(--nx-swatch-indigo-hs), 80%, 0.2),
```
and likewise on the other lines in both light and dark mode